.env.example

```
@@ -1,3 +1,14 @@
-NEXT_PUBLIC_cookieKitId
+
+# Resend API key for email sending
 NEXT_PUBLIC_resend
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The NEXT_PUBLIC_resend key should be with a value or have an equal sign

(KeyWithoutValue)